### PR TITLE
Registering acceptable domains

### PR DESCRIPTION
**Pull request by EDsCODE**
-fixed bug with string matching for accepted domains where the substring comparison wasn't being satisfied and the config file didn't have withsempo at all!

### CODE REVIEW

File: app/server/api/auth_api.py [hunk 1]
```diff
@@ -86,9 +86,11 @@ def post(self):
         tier = None
         sempoadmin_emails = current_app.config['SEMPOADMIN_EMAILS']
```
Comment (michielderoos) [inline app/server/api/auth_api.py]: Does reading config parse the CSV for you?

Comment (EDsCODE) [reply app/server/api/auth_api.py]: yeah, it was getting the values but the condition below `email in sempoadmin_emails` is checking if the entire email is in the accepted domains

Comment (EDsCODE) [reply app/server/api/auth_api.py]: oh wait is this supposed to be working differently than i'm thinking

**Comment by EDsCODE** [comment]
I think i realized my confusion. In trying to restart my entire deployment, I ran into the issue where registering my account wouldn't work because the server would respond with invalid domain. I looked at the PR and noticed you were comparing full emails. @enjeyw with the new PR you did for parsing admin emails, are we supposed to set the config variable ourselves or should we have the email_whitelist table populated when the database initialized during the migrate.py update_data call?

**Comment by EDsCODE** [comment]
@michielderoos clarified the setup. we're good now!

**Pull request closed without merging**